streamingvisitors/src/vespa/searchvisitor/searchvisitor.cpp

```diff
@@ -966,6 +966,15 @@ void SearchVisitor::setupAttributeVector(const FieldPath &fieldPath) {
     }
 }
 
+namespace {
+bool notContained(const std::vector<size_t> sortList, size_t idx) {
```

Comment:
Did you mean to pass by value (and not by reference)?

Reply:
good catch, thanks